src/components/ShowCryptos.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import './showCryptos.css';
+import { updateFilterCryptos } from '../redux/cryptos/filtercryptos';
+import SearchImg from '../assets/images/searchIcon.png';
+
+let txtTo = '';
+const ShowCryptos = () => {
+  const [txtToSearch, setTxtToSearch] = useState('');
+  const cryptosFilterArray = useSelector((state) => state.stFilterCryptos);
+
+  const cryptosArray = useSelector((state) => state.stCryptos);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (txtToSearch === '') {
+      setTxtToSearch(txtTo);
+      document.getElementById('search').value = txtTo;
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const filteredCryptos = cryptosArray
+      .filter((crypto) => crypto.name.toLowerCase().includes(txtToSearch.toLowerCase()));
+    dispatch(updateFilterCryptos(filteredCryptos));
+    txtTo = txtToSearch;
+  };
+
+  const cryptoDiv = cryptosFilterArray.map((crypto) => (
+    <div key={crypto.id} className="oneCryptoDiv">
+      <h1>{crypto.name}</h1>
+      <Link to={`/Details/${crypto.id}`}>
+        <img
+          className="image"
+          src={crypto.image}
+          alt={crypto.name}
+        />
+      </Link>
+      <p>
+        Curr. Price: $
+        {crypto.current_price}
+      </p>
+    </div>
+  ));
+
```

Comment:
- [ ] Kindly make sure each category card uses the expected `text_color`, `text alignment`,.... as on the template design, and remember to add the missing `arrow` on each card to match the expected design. Below is your current design:
![Screenshot from 2022-10-08 03-32-51](https://user-images.githubusercontent.com/83042742/194681403-d622e697-bfa3-4aac-8486-5cbb7672c052.png)

Reply:
@tresorsawasawa Done !